apps/frontend/src/app/features/admin/pages/products/admin-products.component.html

```
@@ -140,73 +156,91 @@ <h5 class="modal-title">{{ product.name }}</h5>
                 </div>
             </div>
-            <div class="modal-footer">
-                <button type="button" class="btn btn-secondary" (click)="closeProductModal()">Cancel</button>
-                <button type="button" class="btn btn-primary" (click)="updateProductStatus()">Update Status</button>
-            </div>
         </div>
-    </div>
-</div>
-}
\ No newline at end of file
+        }
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div></div> ], open tag match failed [ <div class="modal-dialog modal-lg modal-dialog-centered"> ] on line 150.

(tag-pair)